src/unstract/sdk/adapters/llm/exceptions.py

```diff
@@ -1,6 +1,6 @@
 from anthropic import APIError as AnthropicAPIError
 from google.api_core.exceptions import GoogleAPICallError
-from mistralai.exceptions import MistralException
+from mistralai.models import SDKError
```

Comment:
```suggestion
from mistralai.models import SDKError as MistralError
```
In order to avoid name related conflicts / confusions I suggest renaming the imported class